SafetyCulture/ember-typeahead
# [PHNX-3145] Typeahead now removes text after selection

The textbox wasn't clearing the text after it lost focus. Running this next in the run loop fixed this and the text is removed. This solution isn't 100% ideal as you can see a slight text flicker, though it works as expected.
### Testing
- Checkout the frontend PR and run `bower install`.
- Go to a typeahead box and select an item from the list.
- In master the selected item's text should still be in the textbox, in the branch the textbox will be empty.
